common/serial.h

```diff
@@ -133,6 +133,8 @@ class Parser : Looper, StateMachine {
       while (!SA::Connected()) YIELD();
       if (!SA::AlwaysConnected()) {
         STDOUT << "Welcome to ProffieOS " << version << ". Type 'help' for more info.\n";
+        STDOUT << "Serial Commands Wiki Page:\n";
```

Comment:
I'm thinking this should be what the "help" command prints.
Then we can get rid of all the rest of the "help" code.

Reply:
As in...cutting out every printout in Help() functions throughout the OS, and putting them on the Wiki page instead?
Would need to have sections like ifndef DISABLE_DIAGNOSTIC_COMMANDS, then the following will also work, etc...

Reply:
The help command isn't very accurate about specifying which commands are available with which defines anyways.

Reply:
This would mean no internet access=no help. But it might save some memory, which I think is more popular than using the help command .

Reply:
People can always print out the wiki page if they need it.